testing/ncids-css-testing/stories/uswds-native/usa-select/src/usa-select.scenarios.js

```diff
@@ -0,0 +1,13 @@
+module.exports = [
+	{
+		label: 'usa-select default',
+		storyId: 'uswds-components-select--default',
```

Comment:
can you add focus/click selectors to ui elements

Reply:
multiple test variant needs regressions as well - though i'm not sure how we would test multiple selections yet, we might need to chat about this later with the group